docs/App.jsx

```diff
@@ -100,6 +116,38 @@ class App extends React.Component {
           defaultValue="0"
         />
         <br />
+        <label>
```

Comment:
I don't see why there should be a distinction between controlled and uncontrolled. I think the example should just show a controlled component, as it would be confusing using it.